### PR TITLE
Fix new account template

### PR DESCRIPTION
**Pull request by AlbanBedel**
Fix creating accounts with colon in the password (issue #81) and obvious LDAP injection (issue #145).

### CODE REVIEW

File: lib/LDAPUserManager.php [hunk 1]
```diff
@@ -265,6 +265,14 @@ public function ensureAttribute(array &$ldif, string $attribute, string $fallbac
 	}
 
 	public function buildNewEntry($username, $password, $base) {
+		// Make sure the parameters don't fool the following algorithm
+		if (strpos($username, PHP_EOL) !== false)
```
Comment (blizzz) [inline lib/LDAPUserManager.php:269]: please use { curly brackets } even if it is just one (according to our coding styles)

**Comment by AlbanBedel** [comment]
All points from the review have been fixed, can this be merged?

**Comment by blizzz** [comment]
@AlbanBedel a commit does not trigger a notification of Github. So I didn't become aware of your changes until now.  Would you mind to rebase? CI was complaining but the report is too old now and gone.

**Comment by AlbanBedel** [comment]
Rebase done

**Review by blizzz — APPROVED** [review]

**Comment by blizzz** [comment]
thanks @AlbanBedel

**Pull request merged**